compute/cloud-client/src/main/java/compute/CreateWithLocalSsd.java

```diff
@@ -0,0 +1,197 @@
+    }
+
+  }
+
+  // Retrieve the newest image that is part of a given family in a project.
+  // Args:
+  //    project: project ID or project number of the Cloud project you want to get image from.
+  //    family: name of the image family you want to get image from.
+  private static Image getImageFromFamily(String project, String family) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the `imagesClient.close()` method on the client to safely
+    // clean up any remaining background resources.
+    try (ImagesClient imagesClient = ImagesClient.create()) {
+      // List of public operating system (OS) images: https://cloud.google.com/compute/docs/images/os-details
+      return imagesClient.getFromFamily(project, family);
+    }
+  }
+
+  // Create an AttachedDisk object to be used in VM instance creation. Uses an image as the
+  // source for the new disk.
+  //
+  // Args:
+  //    diskType: the type of disk you want to create. This value uses the following format:
+  //        "zones/{zone}/diskTypes/(pd-standard|pd-ssd|pd-balanced|pd-extreme)".
+  //        For example: "zones/us-west3-b/diskTypes/pd-ssd"
+  //
+  //    diskSizeGb: size of the new disk in gigabytes.
+  //
+  //    boot: boolean flag indicating whether this disk should be used as a
+  //    boot disk of an instance.
+  //
+  //    sourceImage: source image to use when creating this disk.
+  //    You must have read access to this disk. This can be one of the publicly available images
+  //    or an image from one of your projects.
+  //    This value uses the following format: "projects/{project_name}/global/images/{image_name}"
+  //
+  //    autoDelete: boolean flag indicating whether this disk should be deleted
+  //    with the VM that uses it.
+  private static AttachedDisk diskFromImage(String diskType, int diskSizeGb, boolean boot,
+      String sourceImage, boolean autoDelete) {
+
+    AttachedDiskInitializeParams attachedDiskInitializeParams =
+        AttachedDiskInitializeParams.newBuilder()
+            .setSourceImage(sourceImage)
+            .setDiskSizeGb(diskSizeGb)
+            .setDiskType(diskType)
+            .build();
+
+    AttachedDisk bootDisk = AttachedDisk.newBuilder()
+        .setInitializeParams(attachedDiskInitializeParams)
+        // Remember to set auto_delete to True if you want the disk to be deleted when you delete
+        // your VM instance.
+        .setAutoDelete(autoDelete)
+        .setBoot(boot)
+        .build();
+
+    return bootDisk;
+  }
+
+  // Create an AttachedDisk object to be used in VM instance creation. The created disk contains
+  // no data and requires formatting before it can be used.
+  // Args:
+  //    zone: The zone in which the local SSD drive will be attached.
+  private static AttachedDisk localSsdDisk(String zone) {
```

Comment:
same comment here, consider `create` or `get` depending on which is more appropriate.

Reply:
Replaced with getLocalSsdDisk()